A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -447,14 +450,34 @@ class A320_Neo_MFD_MainPage extends NavSystemPage {
             this.map.instrument.bingMap.style.visibility = "hidden";
         }
     }
+    // Show/Hide the route
+    setFlightPlanVisibility(flightPlan) {
+        if (flightPlan != this.showFlightPlan) {
+            this.showFlightPlan = flightPlan;
+            this.map.instrument.attributeChangedCallback("show-flightplan", null, flightPlan ? "true" : "false");
+            if (!flightPlan) {
+                this.map.instrument.showConstraints = false;
+                this.map.removeTopOf();
+            } else {
+                const active = SimVar.GetSimVarValue("L:BTN_CSTR_" + this.screenIndex + "_FILTER_ACTIVE", "number");
+                console.log(active);
```

Comment:
Remove `console.log` call

Reply:
This should be removed in the next commit when pushed